wdl-lint/tests/lints/input-not-sorted/source.errors

```diff
@@ -30,6 +30,36 @@ note[InputSorting]: input not sorted
      Int? f = 2
      String g = "hello"
 
+warning[MatchingParameterMeta]: parameter metadata in task `bar` is out of order
```

Comment:
Can you put the parameter meta in the same order as the inputs? Then I would expect this diagnostic to go away, but the `InputSorting` diagnostic to remain.